model/src/main/java/com/ge/predix/acs/model/Obligation.java

```diff
@@ -0,0 +1,97 @@
+package com.ge.predix.acs.model;
+
```

Comment:
package is missing and License add/(run the script)

Reply:
package? without it it won't compile lol

Reply:
added license